python/GafferSceneUI/InteractiveRenderUI.py

```diff
@@ -229,6 +233,100 @@ def __startPauseClicked( self, button ) :
 	def __stopClicked( self, button ) :
 		self.getPlug().setValue( GafferScene.InteractiveRender.State.Stopped )
 
+###############################################################################
+# A widget presenting a summary of the messages in a render nodes messages plug
+###############################################################################
+
+class _MessageSummaryPlugValueWidget( GafferUI.PlugValueWidget ) :
+
+	def __init__( self, plug ) :
+
+		self.__summaryWidget = GafferUI.MessageSummaryWidget( criticalOnly = True, buttonToolTip = "Click to open the render log" )
+		GafferUI.PlugValueWidget.__init__( self, self.__summaryWidget, plug )
+
+		self.__summaryWidget.levelButtonClickedSignal().connect( Gaffer.WeakMethod( self.__levelButtonClicked ), scoped = False )
+
+		self._updateFromPlug()
+
+	def getToolTip( self ) :
+
+		return None
+
+	@GafferUI.LazyMethod( deferUntilPlaybackStops = True )
+	def _updateFromPlug( self ) :
+
+		if self.getPlug() is not None :
+			with self.getContext() :
+				messages = self.getPlug().getValue().value
+		else :
+			messages = Gaffer.Private.IECorePreview.Messages()
+
+		self.__summaryWidget.updateFromMessages( messages )
+
+	def __levelButtonClicked( self, level ) :
+
+		if self.getPlug() is None :
+			return
```

Comment:
Should we reimplement `setPlugs()` to disable the buttons when there is no plug, so we can't even get here?

Reply:
Improved https://github.com/GafferHQ/gaffer/pull/3805/commits/b65911def743f09c48019dd076b24fa3d69da889#diff-6337748cd751cfa6969b07c0aad7d52eR273-R277